### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by fabiankaegy**
Ive added some helpful translations

### CODE REVIEW

File: one-more.md [hunk 1]
```diff
@@ -0,0 +1 @@
+also adding somethin here
```
Comment (rbest) [inline one-more.md]: hey you misspelled something

Comment (fabiankaegy) [reply one-more.md]: I fixed it  👍

Comment (rbest) [reply one-more.md]: cool done

**Comment by fabiankaegy** [comment]
#2

**Pull request merged**